### DISCUSSION TITLE
Update

### DISCUSSION THREAD

**Discussion by gokcenveysel** [chartbrew/chartbrew#112] [Q&A]
Hi,

I want to update from v1.20.2 to 1.22.1 but is there any documentation on how I can do it? Or am I have to re-install again?

**Comment by razvanilin** [comment]
Hey @gokcenveysel 👋 There is currently no documentation on the Update procedure, but I just opened an issue to keep track of this as I should update the docs indeed.

Can you tell me how you installed Chartbrew so that I can help you with the instructions accordingly?

**Reply by gokcenveysel** [reply]
Hi @razvanilin ,

Thank you for your response.

I am cloning the repository and using the following:

```
git clone https://github.com/chartbrew/chartbrew.git
cd chartbrew && npm run setup
```
By the way, install release 1.21.1 source code zip but I don't know how I can implement those files into the current release(1.20.2)

**Reply by razvanilin** [reply]
If you installed it with `git clone`, all you have to do is to pull the latest changes from `master`. You can go in your Chartbrew folder and run this:

```
$ git checkout package-lock.json
$ git checkout client/package-lock.json
$ git checkout server/package-lock.json
$ git pull origin master
$ cd client && npm install
$ cd ../server && npm install
```

If you changed the source code, you might run into merge issues which you have to resolve yourself based on what changes you want in the source files.

Let me know how it goes!

**Reply by gokcenveysel** [reply]
Hi @razvanilin ,

Sorry for my late answer.
Unfortunately, the method you suggested to me to update Chartbrew did not work for me.
No request could be sent to any backend services after updating. It gave 404 errors every time.
I solved the problem by downloading the project from GitHub again and entering the server and client folders and running the "npm install" command.

**Reply by razvanilin** [reply]
Thanks for coming back with more info and glad it works now for you!

Not exactly sure why you were getting the 404 errors since pulling the latest changes from `master` should not change your custom variables. I'll write some proper documentation for updating soon
